spritz/app/proguard-rules.pro

```diff
@@ -0,0 +1,25 @@
+# Add project specific ProGuard rules here.
```

Comment:
Do we need this file?

Reply:
❌ in 4bfb1654368f0becc7a283392e72483f336b173b.